packages/run-it/src/components/RequestForm/RequestForm.spec.tsx

```diff
@@ -339,13 +341,15 @@ describe('RequestForm', () => {
 
     expect(screen.getByRole('textbox')).toBeInTheDocument()
     const input = screen.getByRole('textbox')
-    userEvent.paste(input, 'foo')
-    userEvent.type(input, '{enter}')
+    await userEvent.click(input)
+    await userEvent.paste(input, 'foo')
+    await userEvent.type(input, '{enter}')
```

Comment:
Get rid of the following waitFor now?

Reply:
Had to add the `waitFor` back because the test was failing without it.